README.md

```diff
@@ -16,12 +16,24 @@ SUBCOMMANDS:
 ```
 
 ### Example Usage
-
+#### Sway
 You can use pw-volume to bind multimedia keys to raise, lower, and mute volume.
 For instance, in Sway's config:
 
 ```
 bindsym XF86AudioRaiseVolume exec pw-volume change +2.5%
 bindsym XF86AudioLowerVolume exec pw-volume change -2.5%
 bindsym XF86AudioMute exec pw-volume mute toggle
-```
\ No newline at end of file
+```
+#### Waybar
+```
+"custom/pw-volume": {
+    "exec": "pw-volume status",
+    "return-type": "json",
+    "interval": 3,
```

Comment:
I don't know if we should have it here in the docs, but I typically use `; pkill -RTMIN+8 waybar"` in my keybindings, and add

```
"signal": 8,
```
to refresh the widget.

Reply:
So, replace ` "interval": 3,` with `"signal": 8,`?

Reply:
I was thinking `"interval": once`, and add `"signal": 8` after that.

And the keybindings above could be updated so that they send the signal to waybar every time they're pressed.
Like
```
bindsym XF86AudioRaiseVolume exec "pw-volume change +2.5%; pkill -RTMIN+8 waybar"
```

Reply:
Got it. BTW, how does `pulseaudio` modules updates without `pkill`?